# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
float64 estTimetoFull
float64 estTimeToEmpty
